snakers4_silero-models_stt.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: Silero Speech-To-Text Models
+summary: A set of compact enterprise-grade pre-trained STT Models for multiple languages.
+image: silero_logo.jpg
+author: Silero AI Team
+tags: [audio, scriptable]
+github-link: https://github.com/snakers4/silero-models
+github-id: snakers4/silero-models
+featured_image_1: silero_stt_model.jpg
+featured_image_2: silero_imagenet_moment.png
+accelerator: cuda-optional
+---
+
+```python
+import torch
+import zipfile
+import torchaudio
+from glob import glob
+# see https://github.com/snakers4/silero-models for utils and more examples
+
+device = torch.device('cpu')  # gpu also works, but our models are fast enough for CPU
+model, decoder, utils = torch.hub.load(github='snakers4/silero-models',
```

Comment:
would this require omegaconf and torchaudio  to be installed?

if so, you should add a cell like in https://github.com/pytorch/hub/blob/master/nvidia_deeplearningexamples_waveglow.md#example
which pip installs these extra packages.

that will make the thing instantly run in Google Colab, and is really valuable!

Reply:
Hi,

> would this require omegaconf and torchaudio to be installed?

Yeah, this would 
That is why I needed to include them in your CI environment above
Actually in colab I also needed to install soundfile, as we are using it as a backend for TorchAudio

> if so, you should add a cell like in
> that will make the thing instantly run in Google Colab, and is really valuable!

yeah, this totally makes sense
by the way, you can see a more extended colab version [here](https://colab.research.google.com/github/snakers4/silero-models/blob/master/examples.ipynb)

I will add this shortly